tests/screenshot/rgb.slint

```diff
@@ -0,0 +1,35 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+TestCase := Window {
+    width: 100px;
```

Comment:
In order to save space on the got repository, we should keep small images.
I think a  default size of 64x64 should be enough.

Reply:
done